src/environmentd/src/bin/environmentd/main.rs

```diff
@@ -364,13 +365,18 @@ pub struct Args {
     #[clap(long, env = "DEPLOY_GENERATION", default_value = "0")]
     deploy_generation: u64,
 
+    /// Can be provided in place of both persist_consensus_url and
+    /// timestamp_oracle_url in order to point both at the same backend
+    #[clap(long, env = "METADATA_BACKEND_URL")]
+    metadata_backend_url: Option<SensitiveUrl>,
```

Comment:
I think we probably want to add `conflicts_with` or `conflicts_with_all` so that someone doesn't specify both this and `persist_consensus_url`/`timestamp_oracle_url`.

Reply:
fixed